distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -390,6 +390,15 @@ grant codeBase "file:/security-sts-attributequeryclaimshandler/security-sts-ldap
     permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}attributeMap.properties", "read";
 }
 
+grant codebase "file:/spatial-wfs-v1_1_0-source" {
+    permission java.lang.RuntimePermission "createClassLoader", "*"};
+
+grant codebase "file:/spatial-wfs-v2_0_0-source" {
+    permission java.lang.RuntimePermission "createClassLoader", "*"};
+
+grant codebase "file:/spatial-wfs-v1_0_0-source" {
+    permission java.lang.RuntimePermission "createClassLoader", "*"};
+
```

Comment:
Can you please combine all of these to ```grant codebase "file:/spatial-wfs-v1_0_0-source/spatial-wfs-v2_0_0-source/spatial-wfs-v1_1_0-source" {
    permission java.lang.RuntimePermission "createClassLoader";
}```

There should already be a block that has only `permission java.lang.RuntimePermission "createClassLoader";` and if that is the case (I'm pretty sure it is) these bundles should be added to that block. This is for performance reasons. The startup performance hit for the security manager is a quadratic function of how many of these blocks there are. We don't want to create more of them than we need to.